includes/Plugin.php

```diff
@@ -248,7 +248,7 @@ public static function load( $main_file ) {
 		$config = new JSON_File( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'dist/config.json' );
 		Feature_Flags::set_mode( $config['flagMode'] );
 		Feature_Flags::set_features(
-			new JSON_File( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'feature-flags.json' )
+			( new JSON_File( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'feature-flags.json' ) )->jsonSerialize()
```

Comment:
I was getting a lot of errors because `JSON_File` _wasn't_ doing on-demand loading for me. `Feature_Flags::$features` was not behaving like an `instanceof ArrayAccess` for me, at least during development, causing the `enabledFeatures` list to be empty.

I'll try again and see if something was up with my local environment, but this solved the issue for me 😢

Reply:
Nope, just tried this again and—at least on my Mac—I needed the call to `jsonSerialize` here or `_googlesitekitBaseData.enabledFeatures` is `[]` on page start.

Maybe I'm just dealing with an outdated PHP version or something, but I _just_ ran `composer install` so I'm not sure what's up there.